tests/shellcheck.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -e
+shopt -s globstar nullglob
+
+CHECK=( **/*.sh bin/* )
+IGNORE=( **/*.csh bin/*.md)
+
+for c in "${!CHECK[@]}"; do
+  for i in "${IGNORE[@]}"; do
+    [[ ${CHECK[c]} == "$i" ]] && unset -v 'CHECK[c]'
+  done
+done
+[[ ${#CHECK[@]} -eq 0 ]] && { echo 'no files to check'; exit 0; }
+
+echo '---'
+echo 'check:'
+for c in "${CHECK[@]}"; do
+  echo "  - ${c}"
+done
+echo
+
+docker run -ti -v "$(pwd):$(pwd)" -w "$(pwd)" \
+  koalaman/shellcheck-alpine:v0.4.7 -x "${CHECK[@]}"
```

Comment:
`latest` or `stable` might be a better tag here.  Also not clear if the `-alpine` version is really necessary.

Reply:
the latest gives an error. I think that for these few checks, an older image may be good enough.